{{cookiecutter.repo_name}}/Makefile

```diff
@@ -0,0 +1,46 @@
+.DEFAULT_GOAL := validate
+
+clean:
+	find . -name '*.pyc' -delete
+	coverage erase
+	rm -rf assets
+
+requirements:
+	pip install -qr requirements/local.txt --exists-action w
+
+validate_python: clean
+	DISABLE_MIGRATIONS=True coverage run --branch --source={{cookiecutter.repo_name}} ./manage.py test {{cookiecutter.repo_name}} --settings={{cookiecutter.repo_name}}.settings.test
+	coverage report
+	make quality
+
+quality:
+	pep8 --config=.pep8 {{cookiecutter.repo_name}}
+	pylint --rcfile=pylintrc {{cookiecutter.repo_name}}
+
+validate: validate_python
+
+static:
+	python manage.py collectstatic --noinput -v0
+
+html_coverage:
+	coverage html && open htmlcov/index.html
+
+extract_translations:
+	python manage.py makemessages -l en -v1 -d django
+	python manage.py makemessages -l en -v1 -d djangojs
+
+dummy_translations:
+	cd {{cookiecutter.repo_name}} && i18n_tool dummy -v
+
+compile_translations:
+	cd {{cookiecutter.repo_name}} && i18n_tool generate -v
+
+fake_translations: extract_translations dummy_translations compile_translations
+
+pull_translations:
```

Comment:
Now that I have thought about it more, my fears of accidental pushes are unfounded. I'll add a `push_translations` target.